lib/prefix.h

```diff
@@ -377,9 +377,9 @@ static inline void ipv4_addr_copy(struct in_addr *dst,
 #endif /*s6_addr32*/
 
 /* Prototypes. */
-extern int str2family(const char *);
-extern int afi2family(afi_t);
-extern afi_t family2afi(int);
+extern int str2family(const char *strint);
```

Comment:
`strint`? or `string`?

Reply:
I like strint better but I'll change it

Reply:
strint = stringinteger, sounds really cool :sunglasses: